core/market/src/protocol/discovery.rs

```diff
@@ -216,12 +229,13 @@ impl Discovery {
         msg: UnsubscribedOffersBcast,
     ) -> Result<(), ()> {
         let num_received_ids = msg.offer_ids.len();
-        if !msg.offer_ids.is_empty() {
-            log::trace!(
-                "Received {} unsubscribed Offers from [{}].",
-                num_received_ids,
-                &caller,
-            );
+        log::trace!(
```

Comment:
As far as I remember this log was only for non empty cases, because it was to much spam. But maybe it is not a problem as long it is on trace, so you can ignore this comment if you want.